processing/src/main/java/org/apache/carbondata/processing/datatypes/PrimitiveDataType.java

```diff
@@ -245,6 +312,28 @@ public void parseAndBitPack(ByteBuffer byteArrayInput, DataOutputStream dataOutp
     dataOutputStream.write(v);
   }
 
+  @Override
+  public int parseComplexValue(ByteBuffer byteArrayInput, DataOutputStream dataOutputStream,
+      KeyGenerator[] generator, Boolean[][] complexDictionaryIndentification, int startOffset)
```

Comment:
What is the use of `complexDictionaryIndentification` if not used

Reply:
Currently no required. Removed

Reply:
Removed